res/matrix-org/invite_template.eml

```diff
@@ -13,13 +13,13 @@ Content-Disposition: inline
 
 Hi,
 
-%(sender_display_name)s %(bracketed_verified_sender)shas invited you into a room
+%(sender_display_name)s %(bracketed_verified_sender)shas invited you into a %(room_type_name)s
 %(bracketed_room_name)son Matrix. To join the conversation, either pick a
 Matrix client from https://matrix.org/docs/projects/try-matrix-now.html or use
 the single-click link below to join via Element (requires Chrome, Firefox,
 Safari, iOS or Android)
 
-%(web_client_location)/#/room/%(room_id_forurl)s?email=%(to_forurl)s&signurl=https%%3A%%2F%%2Fmatrix.org%%2F_matrix%%2Fidentity%%2Fapi%%2Fv1%%2Fsign-ed25519%%3Ftoken%%3D%(token)s%%26private_key%%3D%(ephemeral_private_key)s&room_name=%(room_name_forurl)s&room_avatar_url=%(room_avatar_url_forurl)s&inviter_name=%(sender_display_name_forurl)s&guest_access_token=%(guest_access_token_forurl)s&guest_user_id=%(guest_user_id_forurl)s
+%(web_client_location)s/#/room/%(room_id_forurl)s?email=%(to_forurl)s&signurl=https%%3A%%2F%%2Fmatrix.org%%2F_matrix%%2Fidentity%%2Fapi%%2Fv1%%2Fsign-ed25519%%3Ftoken%%3D%(token)s%%26private_key%%3D%(ephemeral_private_key)s&room_name=%(room_name_forurl)s&room_avatar_url=%(room_avatar_url_forurl)s&inviter_name=%(sender_display_name_forurl)s&guest_access_token=%(guest_access_token_forurl)s&guest_user_id=%(guest_user_id_forurl)s&room_type=%(room_type)s
```

Comment:
@t3chguy Any chance you could move the template fix off to a separate PR so at least that's done?

Reply:
(Or I can do it if you would like. 😄 )

Reply:
Happy to

Reply:
Broken out to https://github.com/matrix-org/sydent/pull/377